### ISSUE TITLE
Flask-AppBuilder broken after new update - cannot import name 'SelectFieldBase' from 'wtforms.fields' 

### ISSUE THREAD

**Issue by raimundovidaljunior** [pallets-eco/wtforms#861]
### Actual Behavior
Flask-AppBuilder is broken After removal of SelectFieldBase from the src/wtforms/fields/__init__.py
```python
     from airflow.www import utils as wwwutils
  File "/home/runner/work/dataeng-airflow-dags/dataeng-airflow-dags/venv/lib/python3.9/site-packages/airflow/www/utils.py", line 28, in <module>
    from flask_appbuilder.forms import FieldConverter
  File "/home/runner/work/dataeng-airflow-dags/dataeng-airflow-dags/venv/lib/python3.9/site-packages/flask_appbuilder/__init__.py", line 5, in <module>
    from .api import ModelRestApi  # noqa: F401
  File "/home/runner/work/dataeng-airflow-dags/dataeng-airflow-dags/venv/lib/python3.9/site-packages/flask_appbuilder/api/__init__.py", line 38, in <module>
    from ..baseviews import AbstractViewApi
  File "/home/runner/work/dataeng-airflow-dags/dataeng-airflow-dags/venv/lib/python3.9/site-packages/flask_appbuilder/baseviews.py", line 22, in <module>
    from .forms import GeneralModelConverter
  File "/home/runner/work/dataeng-airflow-dags/dataeng-airflow-dags/venv/lib/python3.9/site-packages/flask_appbuilder/forms.py", line 16, in <module>
    from .fields import EnumField, QuerySelectField, QuerySelectMultipleField
  File "/home/runner/work/dataeng-airflow-dags/dataeng-airflow-dags/venv/lib/python3.9/site-packages/flask_appbuilder/fields.py", line 8, in <module>
    from wtforms.fields import Field, SelectField, SelectFieldBase
ImportError: cannot import name 'SelectFieldBase' from 'wtforms.fields' (/home/runner/work/dataeng-airflow-dags/dataeng-airflow-dags/venv/lib/python3.9/site-packages/wtforms/fields/__init__.py)
```
this is in consequence breaking a few airflow packages like so 
```python 
from airflow.www import utils as wwwutils
``` 

this started happening today so I assume it was due to the new release, more specifically [this](https://github.com/pallets-eco/wtforms/commit/f6cc1f5192263c64d06fededd2e06ed654eabb1e#diff-fbdbef46c5e49b751bee71beb0df17cf0abf8cfd7c5cd4fe1b27f880a809ef50L2)
### Expected Behavior
expected behaviour is having Flask-AppBuilder work properly
### Environment

* Python version: 3.9.12
* wtforms version: 3.2.0

**Comment by azmeuk** [comment]
Thank you for your report. Can you confirm that #862 solves your import issues?

**Comment by potiuk** [comment]
Just to confirm. Our CI in Airflow started to detect it as well https://github.com/apache/airflow/actions/runs/11435317651/job/31810505554#step:16:24

I am going to test the fix  now.

**Comment by azmeuk** [comment]
The fix is in the 3.2.1 release.

**Comment by potiuk** [comment]
Thank you. Confirmed it solved our CI.
